clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -23,12 +23,16 @@
   // Version 2 is the same as version 1.
   //
   // Version 3 is the first flexible version.
-  "validVersions": "0-3",
+  //
+  // Version 4 adds CoordinatorKeys
```

Comment:
nit: Could we add the KIP number here? I would also explain a bit more the change. The comment you have in the response is much better for instance.